### DISCUSSION TITLE
Support for Mojo

### DISCUSSION THREAD

**Discussion by kv9898** [posit-dev/positron#5013] [Feedback] [answered]
Although Mojo is also at a very early stage (just like Python), it seems like the next big thing in data science given that it's a superset of Python. Does Positron consider adding support for Mojo in the future? It seems that much existing work for Python can be migrated so I guess the workload is not overwhelming?

**Comment by juliasilge** [comment, marked as answer]
Positron is in fact built with the kind of extensibility that would allow us to add support for Mojo, or other data science languages that become important in the future. We are up for collaborating with folks who are interested in building on Positron's API for extensions, but we do want to highlight that this API is not fully mature yet and is still changing fairly rapidly.
